packages/core/src/domain/configuration.spec.ts

```diff
@@ -62,22 +62,22 @@ describe('configuration', () => {
   describe('sdk_version, env, version and service', () => {
     it('should not modify the logs and rum endpoints tags when not defined', () => {
       const configuration = buildConfiguration({ clientToken }, usEnv)
-      expect(configuration.rumEndpoint).toContain(`&ddtags=sdk_version:${usEnv.sdkVersion}`)
+      expect(decodeURIComponent(configuration.rumEndpoint)).toContain(`&ddtags=sdk_version:${usEnv.sdkVersion}`)
```

Comment:
Decoding the whole URL with `decodeURIComponent` seems weird to me. Could we avoid it by looking for encoded strings instead?

```suggestion
      expect(configuration.rumEndpoint).toContain(`&ddtags=sdk_version%3A${usEnv.sdkVersion}`)
```

Else, we might want to add a test to check if tags are correctly encoded.

Reply:
having the encoded strings make the assertions not so lisible, I'll add another test on tags correctly encoded